test/fixtures/rules/std_rule_cases/L010.yml

```diff
@@ -35,3 +35,26 @@ test_fail_capitalisation_policy_capitalise:
     rules:
       L010:
         capitalisation_policy: capitalise
+
+test_fail_date_part_inconsistent_capitalisation:
+  # Test that time unit capitalization is fixed
+  fail_str: SELECT dt + interval 2 day, interval 3 HOUR
+  fix_str: SELECT dt + INTERVAL 2 DAY, INTERVAL 3 HOUR
```

Comment:
Could you add a pass unit test as well please?
```
test_pass_date_part_consistent_capitalisation:
  # Test that correctly capitalized time units are left unchanged.
  pass_str: SELECT dt + INTERVAL 2 DAY, INTERVAL 3 HOUR
```
(I'm sure it works but always good to check we don't raise false positives in the tests)

Reply:
Added (with small change: removed `.` from end of comment to match file's current file).

Btw there were no other `pass_str` tests in `L010.yml` so this is the first one. I suppose you noticed.

Reply:
Yeah I saw, some of these tests are fairly old. The more recent rules have more in-depth test cases so best to be thorough with anything new we add. No harm in building out the test cases of old rules in a separate PR if you wanted though 👍